.github/workflows/build-scala-cli-template/chisel-template.scala

```diff
@@ -1,11 +1,11 @@
 //> using repository "sonatype-s01:snapshots"
 //> using scala "2.13.12"
-//> using lib "org.chipsalliance::chisel::@VERSION@"
+//> using dep "org.chipsalliance::chisel::@VERSION@"
 //> using plugin "org.chipsalliance:::chisel-plugin::@VERSION@"
 //> using options "-unchecked", "-deprecation", "-language:reflectiveCalls", "-feature", "-Xcheckinit", "-Xfatal-warnings", "-Ywarn-dead-code", "-Ywarn-unused", "-Ymacro-annotations"
 
-import chisel3._
 import circt.stage.ChiselStage
+import chisel3._
```

Comment:
People are most likely to add `import chisel3.util._` right below `import chisel3._`, this makes it less likely that someone ends up with imports like:
```scala
import chisel3._
import chisel3.util._
import circt.stage.ChiselStage
```
Which leads the compiler to complain that `stage` is not a member of package `chisel3.util.circt`

Reply:
Consider using `_root_` here to keep things alphabetic.

Reply:
so instead of
```scala
import circt.stage.ChiselStage
import chisel3._
```
Are you suggesting?
```scala
import chisel3._
import _root._circt.stage.ChiselStage
```
I don't hate it. I don't care that much about alphabetical order but I do want to try to help users avoid the common `object stage is not a member of package chisel3.util.circt` confusion.

What do you prefer?

Reply:
sorry what is the underlying issue? Is the rule that if you do a wildcard import chisel always assumes that the prefix for anything after that?? Or is the issue that we have both a `_root` circt package and a `chisel3.circt` package? (or and a chisel3.util.circt` package?

Reply:
> if you do a wildcard import chisel always assumes that the prefix for anything after that?

Not Chisel, the Scala compiler, the problem being if you have

```scala
import chisel3.util._
import circt.stage.ChiselStage
```
Then because `chisel3.util.circt` exists, it then looks for `chisel3.util.circt.stage` which does not.

FWIW I am not in this spot trying to discuss ways to resolve that more fundamental issue. This is a problem in existing versions so I at least want to tweak our examples to guide users away from hitting it.

Reply:
i think either is fine, but can we add a comment above wherever we put the ChiselStage import to this effect

Reply:
I think using `_root_` is probably best, but I also think adding a comment would be helpful.

Reply:
`_root_` with comment coming right up!